### PR TITLE
Feature/ekb responsive portfolio/2

### PR DESCRIPTION
**Pull request by buelowek1225**
Addressed previous comments from pull request: hex color capitalization, removed commented out code, left my set up with only one css file for this project. In the future I will practice working with multiple css files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+
+**Project:** Create a Resposive Portfolio
+
+**Goal of this Project:** Practice setting up a responsive website without using bootstrap.
+
+**Created by:** Ellen Buelow
+
+**Directions:** Please feel free to test the responsiveness of my profile site. The different layouts at which my site is tested include: 980px, 768px, and 640px.
+
+The layout px size was correspondes to a typical desktop layour (980px), tablet (768px), and phone landscape (640px).
```
Comment (wrobe0709) [inline README.md]: `layout` instead of `layour`

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: end;
+  padding-top: 10px;
+  padding-right: 10px;
+}
+
+li {
+  display: inline;
```
Comment (wrobe0709) [inline css/style.css:29]: Do you need this?

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+  color: #f5f5f5;
+  background-color:#e1e1e166;
+  padding: 5px 0px;
+  text-align: center;
+}
+
+.profile_pic {
+  width: 191px;
+  height: 261px;
+  margin-bottom: 0px;
```
Comment (wrobe0709) [inline css/style.css]: You could shorten this to `margin: 0 10px`

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+  margin-right: 10px;
+  margin-left: 10px;
+  align-content: left;
+  display: inline;
+  float: left;
+}
+.profile_discrip {
+  display: inline;
+  margin: 1%;
```
Comment (wrobe0709) [inline css/style.css]: I think you should stick to using all `px` or all `%`.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+  text-align: left;
+  margin-left: 20%;
+  font-weight: bold;
+  color: #2b344a;
+  font-family: 'K2D', sans-serif;
+  margin-top: 5px;
+}
+
+.form-input {
+  font: unset;
```
Comment (wrobe0709) [inline css/style.css]: Do you need to do this?

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+  background-color: #e1e1e166;
+  color: #2b344a;
+  font-family: 'K2D', sans-serif;
+  font-size: 13px;
+}
+
+.form-submit {
+  background-color: #e1e1e166;
+  color: #2b344a;
+  font-family: unset;
```
Comment (wrobe0709) [inline css/style.css]: I don't think you need to do the `unset`.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+}
+
+/* Portfolio Page */
+
+.column {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
```
Comment (wrobe0709) [inline css/style.css]: Remove extra space.

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+  flex-wrap: wrap;
+  width: 100%;
+  align-content: center;
+  margin: auto;
+  justify-content: center;
+}
+
+.row-item {
+  margin: 4%
+
```
Comment (wrobe0709) [inline css/style.css]: Remove extra space

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+  }
+}
+
+@media screen and (max-width:768px){
+  .personal{
+    margin-bottom: 50px;
+  }
+}
+
+@media screen and (max-width:640dpx){
```
Comment (wrobe0709) [inline css/style.css]: Typo in `max-width`. Should be `640px`.

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,232 @@
...
+  left: 20px;
+  width: 25px;
+}
+
+.siteLink {
+  font-weight: bold;
+  color: #2b344a;
+}
+
+@media screen and (max-width:980px){
```
Comment (wrobe0709) [inline css/style.css]: Space after `:`

File: index.html [hunk 11]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```
Comment (wrobe0709) [inline index.html:4]: In this file you use 2 space for tabs in the others you use 4. I think you should use one or the other.

File: portfolio.html [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="footer">
+      <li>contact: buelowek1225@gmail.com</li>
+    </div>
+  </div>
+</body>
+</html>
```
Comment (wrobe0709) [inline portfolio.html]: Add a newline to end of the file.

**Review by wrobe0709 — CHANGES_REQUESTED** [review]
Review comments have been added to the PR.

File: css/style.css (hunk 2 above)
Comment (buelowek1225) [reply css/style.css:29]: Yes, they align the links in my navbar horizonally.

File: css/style.css (hunk 3 above)
Comment (buelowek1225) [reply css/style.css]: Thanks for the tips.

File: css/style.css (hunk 4 above)
Comment (buelowek1225) [reply css/style.css]: Turns out this code is irrelevant, I removed it. I will keep this in mind moving forward.

File: css/style.css (hunk 5 above)
Comment (buelowek1225) [reply css/style.css]: I removed the unset, I don't need it.

File: css/style.css (hunk 6 above)
Comment (buelowek1225) [reply css/style.css]: Correct, it has been removed.

File: css/style.css (hunk 7 above)
Comment (buelowek1225) [reply css/style.css]: Corrected, space removed

File: css/style.css (hunk 8 above)
Comment (buelowek1225) [reply css/style.css]: Space removed

File: css/style.css (hunk 9 above)
Comment (buelowek1225) [reply css/style.css]: Corrected

File: css/style.css (hunk 10 above)
Comment (buelowek1225) [reply css/style.css]: Fixed typo on each max width line.

File: index.html (hunk 11 above)
Comment (buelowek1225) [reply index.html:4]: The are all 2.

File: portfolio.html (hunk 12 above)
Comment (buelowek1225) [reply portfolio.html]: Lines changes to include one line at the end of each file.

**Review by wrobe0709 — APPROVED** [review]
